ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Crypto/DSIGN.hs

```diff
@@ -112,8 +103,25 @@ instance DSIGNAlgorithm ByronDSIGN where
           then Right ()
           else Left "Verification failed"
 
-    abstractSizeVKey _ = error "Ouroboros.Consensus.Byron.Crypto.DSIGN: DSIGNAlgorithm ByronDSIGN"
-    abstractSizeSig  _ = error "Ouroboros.Consensus.Byron.Crypto.DSIGN: DSIGNAlgorithm ByronDSIGN"
+    seedSizeDSIGN _ = 32
```

Comment:
It would be nice if we could run [`testDSIGNAlgorithm`](https://github.com/input-output-hk/cardano-base/blob/9d6e0f5efc8fe1bfff4825b3f52e80cdceac07b0/cardano-crypto-class/test/Test/Crypto/DSIGN.hs#L37) on this instance, but that's not so easy to accomplish (expose a test lib in `cardano-base`?)